backend/app.js

```diff
@@ -3,12 +3,16 @@ var express = require('express');
 var path = require('path');
 var cookieParser = require('cookie-parser');
 var logger = require('morgan');
-var studentRouter = require('./routes/studentRouter');
+
+var session = require('express-session');
```

Comment:
Please make these in alphabetical order and separate custom and npm imports.

Reply:
Apart from the ones that have to be called in a particular order I organized them in an alphabetical order